### ISSUE TITLE
BaseReportingFragment.java:88: 'BaseReport.getState()' on a null object reference 

### ISSUE THREAD

**Issue by yongjhih** [Instabug/Instabug-Android#313]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'com.instabug.library.model.State com.instabug.library.model.BaseReport.getState()' on a null object reference (BaseReportingFragment.java:88)
```

### SDK Version

8.7.0

### Android Version
OS Level 29

### Device Model
samsung SM-G975U1

**Comment by mzelzoghbi** [comment]
Hello @yongjhih , Thanks for reporting this.
This issue has been resolved on latest versions, please update to `9.1.5` and let us know if you encountered any further issues.

Thank You!

**Comment by yongjhih** [comment]
Thanks for your quick reply.
